src/sonic-bgpcfgd/tests/test_bbr.py

```diff
@@ -185,7 +186,32 @@ def test___init_8():
             "bbr": expected_bbr_entries,
         }
     }
-    __init_common(constants, "BBRMgr::Initialized and enabled. Default state: 'enabled'", None, expected_bbr_entries, "enabled")
+    __init_common(constants, "BBRMgr::Initialized and enabled from constants. Default state: 'enabled'", None, expected_bbr_entries, "enabled")
+
+@patch('bgpcfgd.managers_bbr.BBRMgr.get_bbr_status_from_config_db', return_value='disabled')
+def test___init_with_config_db_1(mocked_get_bbr_status_from_config_db):
```

Comment:
@Gfrom2016 suggest to use a meaningful case name, such as test___init_with_config_db_default_disable

Reply:
refactor testcase name